README.md

```diff
@@ -502,6 +503,38 @@ Supported error codes:
 * `pb_utils.TritonError.UNAVAILABLE`
 * `pb_utils.TritonError.UNSUPPORTED`
 * `pb_utils.TritonError.ALREADY_EXISTS`
+* `pb_utils.TritonError.CANCELLED` (since 23.10)
+
+#### Request Cancellation
+
+One or more requests may be cancelled during execution, for example, cancelled
+by the user. Starting from 23.10, `request.is_cancelled()` returns up-to-date
```

Comment:
Just a reminder, when reviewing release branch for 23.11, make sure 23.10 here is not replaced with 23.11.